src/sys/macos.rs

```diff
@@ -66,10 +61,16 @@ impl Mouse {
     pub fn move_to(&self, x: i32, y: i32) -> Result<(), Box<dyn error::Error>> {
         let point = CGPoint::new(x as _, y as _);
 
-        match unsafe { CGDisplayMoveCursorToPoint(CGMainDisplayID(), point) } {
-            0 => Ok(()),
-            err => Err(Box::new(Error::CGDisplayMoveCursorToPoint(err))),
-        }
+        CGEvent::new_mouse_event(
+            Self::event_source()?,
+            CGEventType::MouseMoved,
+            point,
+            CGMouseButton::Left, // ignored
+        )

```

Comment:
Disclaimer: I dont have any idea how core graphics work in MacOS

Is there a reason why this parameter is required for a move?